dbloader/riak_loader.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import gevent
+from gevent.pool import Pool
+import json
+import random
+import riak as r
+import time
+from . import Loader
+from . import logger
+
+
+class RiakLoader(Loader):
+    ''' Class for load testing Riak. '''
+
+    def __init__(self):
+        '''
+        Initialize a RiakLoader
+        '''
+        Loader.__init__(self)
+        self.dbtype = 'Riak'
+        self.databases = ['rb_1', 'rb_2', 'rb_3']
+        self.db_prefix = 'riak_load_'
```

Comment:
Is there a need for db_prefix? we are not using it anywhere in the loader.

Reply:
correct - we don't need it